src/containers/smartContractComposer/components/contractAddressValidation.tsx

```diff
@@ -565,7 +563,13 @@ const ContractAddressValidation: React.FC<Props> = props => {
                   <Link
                     external
                     type="neutral"
-                    iconRight={<IconLinkExternal height={13} width={13} />}
+                    iconRight={
+                      <Icon
+                        icon={IconType.LINK_EXTERNAL}
+                        height={13}
+                        width={13}
```

Comment:
The component does not support the `height` and `width` attributes but 3 different sizes: 
sm => 12px
md => 16px
lg => 20px
For this case we can probably just use `size="sm"`: 
```suggestion
                        size="sm"
```
The same applies to all the other `height` and `width` usage

Reply:
Thanks for pointing out, fixed!